src/excalidraw-app/index.tsx

```diff
@@ -626,9 +626,9 @@ const ExcalidrawWrapper = () => {
         <MainMenu.DefaultItems.Socials />
         <MainMenu.Separator />
         <MainMenu.DefaultItems.ToggleTheme />
-        <div className="dropdown-menu-item-base" style={{ margin: "0.5rem 0" }}>
+        <MainMenu.ItemCustom style={{ marginTop: "0.5rem" }}>
```

Comment:
I'm still not happy about this inline style tho

Reply:
Now you should be happy 😊